java/ql/src/utils/modeleditor/ApplicationModeEndpoints.ql

```diff
@@ -0,0 +1,28 @@
+/**
+ * @name Fetch endpoints for use in the model editor (application mode)
+ * @description A list of 3rd party endpoints (methods) used in the codebase. Excludes test and generated code.
+ * @kind problem
+ * @problem.severity recommendation
```

Comment:
That's almost certainly incorrect meta-data. I think you can just use `@kind table` and remove the `@problem.severity` line.